packages/marshal/src/marshal.js

```diff
@@ -345,8 +345,14 @@ function assertCanBeRemotable(val) {
   assert(!Array.isArray(val), X`Arrays cannot be pass-by-remote`);
   assert(val !== null, X`null cannot be pass-by-remote`);
 
-  const keys = ownKeys(val);
+  const descs = getOwnPropertyDescriptors(val);
+  const keys = ownKeys(descs); // enumerable-and-not, string-or-Symbol
   keys.forEach(key => {
+    // @ts-ignore
+    assert(
+      !descs[key].get,
```

Comment:
Not a good test. An accessor property with only a setter and no getter with have a descriptor with `get: undefined` which is falsy and so will illegitimately pass this test.
```suggestion
      !('get' in descs[key]),
```

Reply:
added